control-plane/config/webhook/manifests.yaml

```diff
@@ -50,6 +50,27 @@ webhooks:
     resources:
     - ingressgateways
   sideEffects: None
+- admissionReviewVersions:
+  - v1beta1
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /mutate-v1alpha1-jwtprovider
```

Comment:
other examples in this repo seem to have this as plural. I am sure it doesn't matter but if you take the code suggestion above then you might need to update this too

Reply:
These are generated from the kubebuilder annotations, and some are plural but many aren't (ingress gateway, service-router, service-resolver, mesh, etc).